packages/react-log-viewer/src/LogViewer/LoggerContext.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { createContext, useState, useContext } from 'react';
+
+export const useLoggerContext = () => {
+  return useContext(LoggerContext);
+};
+
+export interface LoggerContextInterface {
+  searchedInput: string | null | undefined;
+  rowInFocus: number | null | undefined;
+  foundWordIndex: number | null | undefined;
+  searchedWordIndexes: Array<number | null | undefined>;
+  highlightedRowIndexes: Array<number | null | undefined>;
+  currentDataSource: number | null | undefined;
+  dataSourceTitles: Array<string | null | undefined>;
+  setCurrentDataSource?: (index: number) => void;
+  setDataSourceTitles?: (dataTitles: Array<string | null>) => void;
+  setRowInFocus?: (index: number) => void;
+  setSearchedInput: (input: string) => void;
+  setFoundWordIndex?: (index: number) => void;
+  setHighlightedRowIndexes: (indexes: Array<number>) => void;
+  setSearchedWordIndexes: (indexes: Array<number>) => void;
+};
+
+const LoggerContext = createContext<LoggerContextInterface | null>(null);
+
+export const LoggerContextProvider = ( children:React.ReactNode) => {
```

Comment:
Change to `({ children:React.ReactNode })`

Reply:
it's currently up because need some eyes to get it building. 

The suggested children:ReactNode causes the issue in the picture.

Reply:
Unfortunately only very keen eyes can get this building without checking it out locally. You can schedule time with a Typescript expert (@kmcfaul, @jschuler , or me) who can peer program with you to get it there.

Reply:
Did you update the syntax to include the brackets?

You might also try changing the type to `React.ReactElement`, or defining the states and `LoggerContext.Provider` inside `LogViewer` instead of using the exported const.

Reply:
Thanks for the advice/offer @redallen. I had also tried with the brackets prior to the pr but the errors were different. `{children}:any` seemed to calm it all down and allow me to build, but is it a viable option or should I still define the context in the root logger file?